src/main/java/racingcar/generator/RandomNumberGenerator.java

```diff
@@ -0,0 +1,23 @@
+package racingcar.generator;
+
+import camp.nextstep.edu.missionutils.Randoms;
+import java.util.function.Supplier;
+
+public class RandomNumberGenerator {
+
+    private static final int STARTING_RANGE = 0;
+    private static final int END_RANGE = 9;
```

Comment:
```suggestion
    private static final int LOWER_BOUND = 0;
    private static final int UPPER_BOUND = 9;
```

저의 짧은 식견으로 의견을 드리자면, 이런식으로 이름을 붙이셔도 좋을것 같습니다.

Reply:
그것도 좋고 STRATING_INCLUSIVE, END_INCLUSIVE도 괜찮아 보인다는 생각도 드네요 네이밍은 항상 어려운것 같아요 ㅠ